### PR TITLE
implement template notebook experiment (#127)

### PR DESCRIPTION
**Pull request by caitlinkermen**
Co-authored-by: NLP <nlp@constellationanalytics.com>

# Description

Within the Structured experiments folder I Renamed folder 2022.02.01.template to 2022.02.01.train to fit in with

Removed the following text from line 106 in the README.md within the Structured Experiments folder
``docker exec -ti scribble-ocr-train bash`

## Type of change

- [ X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [X ] Not necessary

## Checklist

- [X ] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [X ] I have commented my code, particularly in hard-to-understand areas
- [X ] I have made corresponding changes to the documentation
- [X ] My changes generate no new warnings
- [X ] I have added tests that prove my fix is effective or that my feature works
- [X ] New and existing unit tests pass locally with my changes
- [X ] There are no merge conflicts

https://www.loom.com/share/5366ec5f086e4badb4e85e2929fa2bda

### CODE REVIEW

**Comment by ashtilawat** [comment]
Loom video?

**Comment by caitlinkermen** [comment]
> 
https://www.loom.com/share/5366ec5f086e4badb4e85e2929fa2bda
I added the link to the description above also

File: structured_experiments/2022.04.28.notebook_template/experiment.MD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+TODO: Create Template
+
+
+
+# Reproduce
+
+## 1. First Acquire the training data
+
+### 
+Run `git lfs pull --exclude .git -I  data/storysquad-ground-truth/`
```
Comment (nlittlepoole) [inline structured_experiments/2022.04.28.notebook_template/experiment.MD:24]: this needs to be changed to tell the user to run `git lfs pull --exclude .git -I data/storysquad-ground-truth/` and `git lfs pull --exclude .git -I data/kaggle-ground-truth/`

**Review by nlittlepoole — CHANGES_REQUESTED** [review]
Should be good after this request

**Comment by nlittlepoole** [comment]
@caitlinkermen you opened this pull request in reverse. 

```
caitlinkermen wants to merge 1 commit into bug/structured-experiment-readme-and-folder-name-issue from main
````

you want into `main` from `bug/structured-experiment-readme-and-folder-name-issue`

**Pull request closed without merging**